modules/phpfpm/client.go

```diff
@@ -128,7 +121,8 @@ func (c httpClient) fetchStatus(req *http.Request) (*status, error) {
 }
 
 func (c socketClient) Status() (*status, error) {
-	socket, err := fcgiclient.Dial("unix", c.socket)
+	var timeout = time.Duration(1) * 100000000
```

Comment:
there is user configurable timeout

https://github.com/netdata/go.d.plugin/blob/5982f13329a41a0f1235afa9189436e196ae2c9c/modules/phpfpm/phpfpm.go#L24

Reply:
Sorry, totaly missed that. i'll add that in the next commit! Thanks!

Reply:
it is `timeout` option in the config

https://github.com/netdata/go.d.plugin/blob/5982f13329a41a0f1235afa9189436e196ae2c9c/config/go.d/phpfpm.conf#L89-L92